tests/utils.py

```diff
@@ -158,7 +158,8 @@ def fill_in_config_options(config_template_path,
                             'suffix'],
                   'Tuning': ['grid_search', 'objective',
                              'use_folds_file_for_grid_search', 'grid_search_folds',
-                             'param_grids', 'objectives', 'duplicate_option'],
+                             'pos_label_str', 'param_grids', 'objectives',
```

Comment:
Why is `pos_label_str` in the tuning section? I always thought it was a bit of bad fit. I usually assume it's in the input section.